.travis.yml

```diff
@@ -1,17 +1,20 @@
 language: python
 python:
-  - "2.7"
-  - "3.4"
+  - "3.6"
+
 install:
-    - pip install -r requirements.txt
-    - pip install braintree coveralls mock mysqlclient
+    - pip install -r requirements.txt   
 env:
   - DB=sqlite
```

Comment:
`sqlite`?

Reply:
#### Why Sqlite?
Locally, we use a SQLite database for testing. So we thought it was good to use the same database system just to confirm that our work was okay.

We had trouble setting up the Postgres DB with Travis. Travis Builds would fail/ error on the database stage. So that is why we had opted for SQLite.

However, we have added Postgres to the list of databases used for purposes of uniformity.